page.codeberg.lazyt.ubpm.yml

```diff
@@ -0,0 +1,27 @@
+app-id: page.codeberg.lazyt.ubpm
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: ubpm
+rename-desktop-file: ubpm.desktop
+rename-icon: ubpm
+finish-args:
+  - --socket=x11
+# wayland fails to open default browser?
+#  - --socket=wayland
+#  - --socket=fallback-x11
+  - --socket=cups
+  - --device=dri
```

Comment:
you probably need `--device=all` here. How does it work with USB?

Reply:
You are right. The USB plugins did not work, I changed it.